ydb/library/yql/minikql/comp_nodes/mkql_wide_combine.cpp

```diff
@@ -1523,6 +1530,10 @@ using TBaseComputation = TStatefulWideFlowCodegeneratorNode<TWideLastCombinerWra
 #endif
 };
 
+bool IsTypeSerializable(const TType* type) {
+    return !type->IsResource();
```

Comment:
As far as I understand there are more non-serializable types. 
Here are all possible types: https://github.com/ydb-platform/ydb/blob/f2efea6e0b234506f42443226951d4944452ce69/ydb/library/yql/minikql/mkql_node.h#L131
And all the types that are not listed here are non-serializable: https://github.com/ydb-platform/ydb/blob/f2efea6e0b234506f42443226951d4944452ce69/ydb/library/yql/minikql/computation/mkql_computation_node_pack.cpp#L167

Also, I think we can use `ctx.Mutables.SerializableValues` to get indexes of serializable values

Reply:
Thank you and I agree with you. Will add.